app/lib/homework/homework_dialog/homework_dialog.dart

```diff
@@ -959,26 +959,32 @@ class _SendNotification extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     final bloc = bloc_lib.BlocProvider.of<HomeworkDialogBloc>(context);
-
     return MaxWidthConstraintBox(
       child: SafeArea(
         top: false,
         bottom: false,
-        child: _SendNotificationBase(
-          title:
-              "Kursmitglieder ${state.isEditing ? "über die Änderungen " : ""}benachrichtigen",
+        child: ListTile(
```

Comment:
I think it's easier to just replace the `ListTileWithDescription` in the `_SendNotificationBase` widget because now the `HwDialogKeys.notifyCourseMembersTile` key is missing.

```suggestion
        child: _SendNotificationBase(
```